lib/k8s/conn/auth/exec_worker.ex

```diff
@@ -0,0 +1,128 @@
+defmodule K8s.Conn.Auth.ExecWorker do
+  @moduledoc false
+  use GenServer
+
+  alias K8s.Conn.Error
+
+  defmodule State do
+    @moduledoc """
+    The state of the exec worker
+    """
+
+    defstruct [:command, :env, args: [], cached_token: nil]
+
+    @type t :: %__MODULE__{
+            command: String.t(),
+            env: %{name: String.t(), value: String.t()},
```

Comment:
I know this is copy pasted, but looking at the code, I think this spec should be for `%{name: "value"}`, right?

```suggestion
            env: %{name: String.t()},
```